lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -289,7 +289,7 @@ protected function _getPortableSequenceDefinition($sequence)
             $sequenceName = $sequence['relname'];
         }
 
-        $data = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
+        $data = $this->_conn->fetchAll($this->_platform->getSequenceMetadataSQL($sequenceName, $sequence['schemaname']));
```

Comment:
Is it really necessary to `fetchAll()` and then use only `$data[0]`? Doing `fetch()` and using the row directly would make more sense to me.

Reply:
There's no method `fetch()` in connection. In my opinion, this is ok as it fetches one row

Reply:
There are more fetch*() methods though, fetchArray() is the one you need.

Reply:
The data then looks like  
![image](https://user-images.githubusercontent.com/327717/34307285-947d3384-e747-11e7-998f-d90a2d9a7083.png). 
I personally think the current state is more verbose as I have to reference array values by column names and not by numeric indeces.